matrix-meetings-widget/src/components/meetings/MemberSelectionDropdown/MemberSelectionDropdown.test.tsx

```diff
@@ -346,4 +346,40 @@ describe('<MemberSelectionDropdown/>', () => {
     );
     await userEvent.click(screen.getByRole('button', { name: '@user-1' }));
   });
+
+  it('should show a message if members could not be loaded', async () => {
+    const { rerender } = render(
+      <MemberSelectionDropdown
+        availableMembers={[]}
+        selectedMembers={[]}
+        label="Members"
+        onSelectedMembersUpdated={jest.fn()}
+        ownUserPopupContent="This is you"
+        loading={true}
```

Comment:
(This is quite minor and I only comment this because I have another comment)

You can (and should) omit the value of boolean properties if it defaults to `false`:

```suggestion
        loading
```

Reply:
changed